samples/UrlHelperSample/wwwroot/web.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These samples are rarely used from VS and my aim here is to get rid of the useless wwwroot directory entirely.